examples/add_job_with_custom_concurrency/main.go

```diff
@@ -29,10 +28,10 @@ func main() {
 		log.Println("got job id:", j.ID, "messsage:", j.Payload["message"])
 		done <- true
 		return
-	}, neoq.HandlerConcurreny(8))
+	}, neoq.HandlerConcurrency(8))
 
 	// Option 2: Set options after the handler is created
-	handler = handler.WithOption(neoq.HandlerConcurreny(8))
+	handler = handler.WithOption(neoq.HandlerConcurrency(8))
```

Comment:
What do you think about changing the `WithOption` method from returning a Neoq interface to `WithOptions(...option)` that returns a void?

Reply:
It does bring down the number of `WithOption` calls...I'll add a followup item to the project board.